package.json

```diff
@@ -1,12 +1,26 @@
 {
-  "name": "myment-front",
-  "version": "1.0.0",
-  "description": "This repo contain the front API for the web application myment.",
-  "main": "index.js",
-  "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
-  },
-  "keywords": [],
-  "author": "",
-  "license": "ISC"
+	"name": "myment-front",
+	"version": "1.0.0",
+	"description": "This repo contain the front API for the web application myment.",
+	"main": "index.js",
+	"scripts": {
+		"test": "echo \"Error: no test specified\" && exit 1",
+		"dev": "parcel src/index.html",
+		"clean": "rm -rf .cache dist"
+	},
+	"alias": {
+		"vue": "./node_modules/vue/dist/vue.esm"
+	},
+	"keywords": [],
+	"author": "",
+	"license": "ISC",
+	"dependencies": {
```

Comment:
The dependencies must be only vue and his dependencies like vue-router or vuex or react

Reply:
I'll bend the knee.